install.sh

```diff
@@ -40,7 +40,12 @@ apt-get install -y  \
   rsync=3.1.0* \
   psmisc=22.20* \
   vim=2:7.4.052*
+  
+#update-rc.d: warning: default stop runlevel arguments (0 1 6) do not match rsync Default-Stop values (none)
```

Comment:
we dont need to paste the warning here. just remove all the warnings and comments for now

Reply:
ok